protocol-units/da-sequencer/config/src/lib.rs

```diff
@@ -6,6 +6,8 @@ use serde::{Deserialize, Serialize};
 use std::net::SocketAddr;
 use tracing::info;
 
+pub const DASEQUENCER_CONF_FOLDER: &str = "da-sequencer";
```

Comment:
"Folder" is not a universal OS terminology, it's a "directory", or "dir",
```suggestion
pub const DA_SEQUENCER_DIR: &str = "da-sequencer";
```